components/log-viewer-webui/server/src/app.js

```diff
@@ -41,6 +42,8 @@ const app = async ({
                 port: settings.MongoDbPort,
             },
         });
+        console.log(settings.StreamFilesS3Region)
+        await server.register(S3Manager, settings.StreamFilesS3Region);
```

Comment:
Then we will be calling this plugin registration with an option object:
```suggestion
        await server.register(S3Manager, {region: settings.StreamFilesS3Region});
```